CSS/CSS.sublime-syntax

```diff
@@ -603,14 +683,16 @@ contexts:
               | hyphens|overflow-scrolling|overflow|color-profile|kerning
               | nbsp-mode|color|image-resolution|grid-row|grid-column|blend-mode
               | azimuth|pause-after|pause-before|pause|pitch-range|pitch|text-height
+              | system|negative|prefix|suffix|range|pad|fallback|additive-symbols
+              | symbols|speak-as
             )\b
           scope: support.type.property-name.css
     - match: (:)\s*
       captures:
         1: punctuation.separator.key-value.css
       push:
-        - meta_scope: meta.property-value.css
-        - match: '\s*(;|(?=\}))'
+        - meta_content_scope: meta.property-value.css
+        - match: '\s*(;)|(?=[})])'
```

Comment:
I'm not too concerned about it breaking highlighting since it isn't valid anyway. If we tried to gracefully handle each language and the common way typos would break syntax we'd have very long and complicated syntaxes.